src/box/lua/console.lua

```diff
@@ -689,6 +724,31 @@ local function client_read(self)
     return buf:sub(1, -#self.delimiter-2)
 end
 
+--
+-- Read a command from connected client console.listen().
+--
+local function client_read(self)
+    local buf = ''
+    local continuation_on = self.continuation_on
+    while true do
+        local line = client_read_line(self)
+        if line == nil then
+            -- EOF or error.
+            --
+            -- Note: Even if `buf` is not empty, skip unfinished
+            -- command.
+            return nil
+        end
+        buf = buf .. line
+        if continuation_on and line:endswith(continuation_symbol) then
+            buf = buf:sub(1, -1 - #continuation_symbol)
+        else
+            break
+        end
```

Comment:
In `local_read()` we handle the backslash only when there is no delimiter. Is there a reason, why we're going another way here, in `client_read()`?

Reply:
It seems to be impossible to set delimiter in net console:
```
local function set_delimiter(_storage, value)
    local console = fiber.self().storage.console
    if console ~= nil and console.delimiter == '$EOF$' then
        return error('Can not install delimiter for net box sessions')
    end
    value = value or ''
    return delimiter(value)
end
```